conf/app.routes

```diff
@@ -11,3 +11,6 @@ POST    /registration/organisation/noId        uk.gov.hmrc.crsfatcaregistration.
 POST    /subscription/create-subscription      uk.gov.hmrc.crsfatcaregistration.controllers.SubscriptionController.createSubscription
 POST    /subscription/read-subscription        uk.gov.hmrc.crsfatcaregistration.controllers.SubscriptionController.readSubscription
 PUT     /subscription/update-subscription      uk.gov.hmrc.crsfatcaregistration.controllers.SubscriptionController.updateSubscription
+
+# org with utr, used for address reteival in FI manaagement
+POST    /registration/v2/organisation/utr      uk.gov.hmrc.crsfatcaregistration.controllers.RegistrationWithUtrController.sendAndRetrieveRegWithUtr
```

Comment:
Can we remove the `v2`? I think either everything should have a version number or nothing should.

Reply:
Aye, it was just to differentiate the endpoint from the other til we had a better name